### PR TITLE
adj legend text size rel to title size

### PR DESCRIPTION
**Pull request by zackarno**
made some tweaks to legend/text parameters. I sent test emails with the 3 possible scenarios for legends to see how it could change. I think they look fine now! Don't worry that the data in the maps doesn't make sense - I was just modifying data to see legend scenarios.

### CODE REVIEW

File: src/gauge_monitoring_email.R [hunk 1]
```diff
@@ -275,8 +276,15 @@ email_creds <- creds_envvar(
   port = Sys.getenv("CHD_DS_PORT"),
   use_ssl = TRUE
 )
-
+email_to <- c(email_to, "seth.caldwell@un.org")
```
Comment (caldwellst) [inline src/gauge_monitoring_email.R]: lol let's not commit this.

File: src/gauge_monitoring_email.R [hunk 2]
```diff
@@ -275,8 +276,15 @@ email_creds <- creds_envvar(
...
 email_rmd_fp <- "email_flood_monitoring.Rmd"
+
+scen_txt <- c(
```
Comment (caldwellst) [inline src/gauge_monitoring_email.R]: And the testing coding doesn't need merging to main.

**Review by caldwellst — CHANGES_REQUESTED** [review]
I think you accidentally committed and pushed the test code, with my email there ha, so definitely remove that and the test text! But then all looks good for merging, and the emails lok good.

**Comment by caldwellst** [comment]
Sorry, just noticed in recent email, can we just add a period to this sentence?

```
Each line represents one monitored basin
```

**Comment by zackarno** [comment]
thank you - good call on all of the above. Fixed

**Review by caldwellst — APPROVED** [review]

**Pull request merged**